### DISCUSSION TITLE
Weird bug

### DISCUSSION THREAD

**Discussion by zerealone** [community/community#19017] [Other Feature Feedback, Questions, & Ideas]
When I went to my lists tab, it showed me the number in the left side, instead of right side and it was at the edge of the screen.
When I refreshed it didn't showed up again , but if it happens next time I'll send here a screenshot.

Device info :
Android, Samsung

Although this bug, the feature is an amazing idea and I started already to use it and it really helped me organize some github projects in other categories.

**Comment by melvincwng** [comment]
Hi @zerealone, you can report bugs to GitHub Support over [here](https://support.github.com/contact/bug-report)

The support engineers usually would get back to you within a few working days, after they investigated the issue you raised.

If this answer has helped you, please kindly 'Mark as answer' so that others know this question has been addressed.

Thanks!

Hope this helps.

**Reply by Patrick2239** [reply]
Ok

**Comment by Patrick2239** [comment]
> When I went to my lists tab, it showed me the number in the left side, instead of right side and it was at the edge of the screen.
> When I refreshed it didn't showed up again , but if it happens next time I'll send here a screenshot.
> 
> Device info :
> Android, Samsung
> 
> Although this bug, the feature is an amazing idea and I started already to use it and it really helped me organize some github projects in other categories.
>

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
